bot/__main__.py

```diff
@@ -28,13 +29,18 @@ def on_error(name: str) -> NoReturn:
         yield module.name
 
 
-if __name__ == "__main__":
+async def start_bot() -> None:
+    """Load in extensions and start running the bot."""
     bot = Friendo(
         command_prefix=settings.COMMAND_PREFIX, help_command=None, intents=Intents.all(),
         allowed_mentions=AllowedMentions(everyone=False),
     )
 
     for cog in _get_cogs():
-        bot.load_extension(cog)
+        await bot.load_extension(cog)
+
+    async with bot:
```

Comment:
`bot` is already in scope, is it necessary to do this?

Reply:
Could you clarify? This is from the discord.py docs, but I am unsure if this is actually the correct way to do this https://discordpy.readthedocs.io/en/stable/migrating.html#asyncio-event-loop-changes

Reply:
I just thought it's kinda weird to create a new async scope when you're in an async function.

Reply:
I agree, it seems quite odd. I am currently unsure what the purpose of this is.

Reply:
Well if it's in the docs, then we should do what the docs say.